tock/projects/migrations/0027_project_include_in_utilization.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.13 on 2020-06-23 16:29
+
+from django.db import migrations, models
+
+def make_default_utilization(apps, schema_editor):
+    Projects = apps.get_model('projects','Project')
+
+    for project in Projects.objects.all():
+        project.include_in_utilization = project.is_billable
```

Comment:
turns out `is_billable` isn't accessible in this context - if you switch this to...

`project.include_in_utilization = project.accounting_code.billable`

...you should be good.